src/AcquiaLiftException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\acquia_lift;
+
+/**
+ * Represents an exception that occurred in some part of the Acquia Lift Module.
+ */
+class AcquiaLiftException extends \Exception {
+}
```

Comment:
I would move this to Exception directory, and namespace it "Drupal\acquia_lift\Exception".